catalog-info.yaml

```diff
@@ -125,3 +125,50 @@ spec:
           access_level: MANAGE_BUILD_AND_READ
         everyone:
           access_level: READ_ONLY
+
+# ***********************************
+# SECTION START: E2E tests pipeline
+# ***********************************
+---
+# yaml-language-server: $schema=https://gist.githubusercontent.com/elasticmachine/988b80dae436cafea07d9a4a460a011d/raw/rre.schema.json
+apiVersion: backstage.io/v1alpha1
+kind: Resource
+metadata:
+  name: logstash-filter-elastic-integration-e2e
+  description: 'A pipeline for `elastic_integration` E2E tests.'
+  links:
+    - title: 'A pipeline for `elastic_integration` E2E tests.'
+      url: https://buildkite.com/elastic/logstash-filter-elastic-integration-e2e
+spec:
+  owner: group:ingest-fp
+  type: buildkite-pipeline
+  system: buildkite
+  implementation:
+    apiVersion: buildkite.elastic.dev/v1
+    kind: Pipeline
+    metadata:
+      description: ':logstash: `elastic_integration` E2E tests. :pipeline:'
+      name: logstash-filter-elastic_integration-e2e
```

Comment:
I recommend we make this name the same as for the resource i.e. `logstash-filter-elastic-integration-e2e` to avoid confusion with things like https://github.com/elastic/logstash-filter-elastic_integration/pull/126/files

Reply:
Definitely agree with your point, updated.
I don't know the risks if we do on other pipelines but I also want to remove `_` from the pipeline definitions beside repo links.

Reply:
Given that the process of slugification converts `_` to `-`, I think we should stick to this convention wherever possible, esp `name:` properties in the catalog-info.